docs/users.md

```diff
@@ -522,6 +601,66 @@ vcluster-0                                          2/2     Running   0
 
 The nginx pod is the one with the name `nginx-x-default-x-vcluster`.
 
+## Working with an external control plane
+
+In this section, we will show an example of creating an external control plane to adopt
+a kind cluster named `ext1`. This setup requires that the external cluster `ext1` 
+and the KubeFlex hosting cluster are on the same docker network.
+
+### Determining the endpoint for an external cluster in the same host machine
+Ensure that both your hosting and adopted clusters are on the same Docker network. For 
+clusters using the default `kind` docker network, execute the following command to 
+check the DNS name of the external cluster `ext1` on the docker network:
+
+```shell
+docker inspect ext1-control-plane | jq '.[].NetworkSettings.Networks.kind.DNSNames' 
+```
+
+The output will show something similar to the following:
+
+```shell
+[
+  "ext1-control-plane",
+  "79540574c3c7"
+]
+```
+The endpoint for the adopted cluster should then be set as follows:
+
+```shell
```

Comment:
What is shown here is not a shell command. In fact, it is so small that I think it would be better to run it inline and just quote it with single backticks.

Reply:
done